d/serverino/source/app.d

```diff
@@ -15,7 +15,7 @@ mixin ServerinoMain;
         .enableKeepAlive(180.seconds)
         .addListener("0.0.0.0", 3000)
         .setWorkers(50)
```

Comment:
This should be lowered down, since now there are multiple daemons, each one with its own workers. So if you set 10 daemon instances, probably you want to set 4 or 5 workers (per daemon)

Reply:
could you suggest any number ?

Reply:
8/4 as you set should be a good starting point I hope :)

Let's see. It's a fresh new feature, not sure what's the right balance.